helm/values-eks.yaml

```diff
@@ -348,35 +343,63 @@ workflow_scheduler:
         reconnectAttempts: 10
         reconnectDelaySeconds: 30
 
-
 # --
 # Workflow notifications module is an optional dependency. Flyte uses cloud native pub-sub systems to notify users of
 # various events in their workflows
 workflow_notifications:
-    enabled: false
-    config:
-       notifications:
-         type: aws
-         region: us-east-2
-         publisher:
-           topicName: arn:aws:sns:us-east-1:<ACCOUNT-NUMBER>:flyte-notifications-topic
-         processor:
-           queueName: flyte-notifications-queue
-           accountId: <ACCOUNT-NUMBER>
-         emailer:
-           subject: "Flyte: {{ project }}/{{ domain }}/{{ launch_plan.name }} has '{{ phase }}'"
-           sender:  "flyte@example.com"
-           body: |
-             Execution {{ workflow.project }}/{{ workflow.domain }}/{{ workflow.name }}/{{ name }} has {{ phase }}.
-             Details: https://flyte.example.com/console/projects/{{ project }}/domains/{{ domain }}/executions/{{ name }}.
-             {{ error }}
-
+  enabled: false
+  config:
+    notifications:
+      type: aws
+      region: us-east-2
+      publisher:
+        topicName: arn:aws:sns:us-east-1:<ACCOUNT-NUMBER>:flyte-notifications-topic
+      processor:
+        queueName: flyte-notifications-queue
+        accountId: <ACCOUNT-NUMBER>
+      emailer:
+        subject: "Flyte: {{ project }}/{{ domain }}/{{ launch_plan.name }} has '{{ phase }}'"
+        sender: "flyte@example.com"
+        body: |
+          Execution {{ workflow.project }}/{{ workflow.domain }}/{{ workflow.name }}/{{ name }} has {{ phase }}.
+          Details: https://flyte.example.com/console/projects/{{ project }}/domains/{{ domain }}/executions/{{ name }}.
+          {{ error }}
 
 # -- Configuration for the Cluster resource manager component. This is an optional component, that enables automatic
 # cluster configuration. This is useful to set default quotas, manage namespaces etc that map to a project/domain
 cluster_resource_manager:
   # -- Enables the Cluster resource manager component
   enabled: true
+  # -- Configmap for ClusterResource parameters
+  config:
+    # -- ClusterResource parameters
+    # Refer to the [structure](https://pkg.go.dev/github.com/lyft/flyteadmin@v0.3.37/pkg/runtime/interfaces#ClusterResourceConfig) to customize.
+    cluster_resources:
```

Comment:
this is already part of the default cluster resource config right?
https://github.com/flyteorg/flyte/blob/helm-expose-cluster-resources/helm/values.yaml#L627